src/vue-resource-case-converter.js

```diff
@@ -46,42 +54,50 @@ function convertObjectKeys(obj, keyConversionFun) {
   }, Array.isArray(obj) ? [] : {}); // preserve "arrayness"
 }
 
+function mergeOptions(globalOptions, localOptions) {
+  // Combines the value of requestUrlFilter/responseUrlFilter with the global options
+  if (typeof(localOptions) === 'boolean') {
+    return Object.assign({}, globalOptions, {convert: localOptions});
```

Comment:
I wouldn't use `Object.assign` here, because it is not supported in IE and Safari 8 which is still used by people. There is a ponyfill [here](https://www.npmjs.com/package/object-assign).